src/main/java/com/foo/unmls/UnmlsGraphTool.kt

```diff
@@ -0,0 +1,41 @@
+package com.foo.unmls
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.int
+
+object UnmlsGraphTool {
+    @JvmStatic
+    fun main(args: Array<String>) {
```

Comment:
For iterative development you might set up an IDE "run configuration" that runs this class with your chosen parameters. Or, if you're CLI focused, you might:

```
./gradlew run --args='foo --widget-size 1'
```

Or if you don't want gradle in the way, but just want to work directly with the jar:

```
./gradlew shadowJar
java -jar build/libs/umls-to-graph-all.jar foo --widget-size=1
```

Or to get fancier...

In one terminal, run gradle in "watch mode". Whenever input files change, it will redo the necessary build steps to generate the shadow jar constantly, so it will keep running until you kill it. Also note that we are abbreviating `shadowJar` as `shJ`, and gradle is ok with that since shJ is an unambiguous abbreviation. `c` would not be, since it could be `clean` or `check`, but `ch` (for `check`, which runs tests) would be ok.
```
./gradlew shJ --continuous
```

In another:
```
java -jar build/libs/umls-to-graph-all.jar foo --widget-size=1
```